src/__test__/CoinCard.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
- [ ] Let's make sure we have all the tests passing. 
<img width="284" alt="Capture" src="https://user-images.githubusercontent.com/58504655/137173403-c63ef1ce-7768-460b-aa21-964e5d1adfbb.PNG">